src/index.js

```diff
@@ -10,7 +10,7 @@ form.addEventListener('submit', (e) => {
 
```

Comment:
![Missing-commands](https://user-images.githubusercontent.com/93039359/196643466-6e0473e9-7f9e-4532-8e27-aca5b8bb2568.PNG)

- [ ] In your setup section you have made it professional but i guess you forgot to add commands of how to clone your repository 😄 No problem you can add them now 😊

Reply:
Thank you for notifying me on this, I have done it now